### PR TITLE
Add pycomlink

### PR DESCRIPTION
**Pull request by cchwala**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/pycomlink/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/pycomlink/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 15b8b5dc4d5ad9e44e06baf58afbde00aa1432ae4fa697b3907df9cdce94a3da
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```
Comment (jakirkham) [inline recipes/pycomlink/meta.yaml]: We can probably lighten the options passed here (`conda-build` handles many of these for us, except `-v`).

File: recipes/pycomlink/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    - pykrige
+    - future
+
+test:
+  imports:
+    - pycomlink
+
+about:
+  home: http://github.com/pycomlink/pycomlink
+  license: BDS-3-Clause
```
Comment (jakirkham) [inline recipes/pycomlink/meta.yaml]:
```suggestion
  license: BSD-3-Clause
```

File: recipes/pycomlink/meta.yaml (hunk 1 above)
Comment (cchwala) [reply recipes/pycomlink/meta.yaml]: Thanks for the quick review.

I took these from another package which has similar dependencies to mine to be on the save side.

I will go back to the suggested pip options from the example.

Comment (jakirkham) [reply recipes/pycomlink/meta.yaml]: Of course. 😄

Yep, that's generally a good idea. The docs and examples here are likely more up-to-date though. For things not included in either, taking things from other recipes is a good plan. 👍

**Comment by jakirkham** [comment]
Thanks @cchwala and welcome to conda-forge! 😄

**Pull request merged**